Module 3

PrivetAdel opened the pull request.

PrivetAdel committed:
feat(components): подключить сервер

проверяю авторизацию на сервере

PrivetAdel committed:
feat(components): переписать роутинг

Переписала роутинг с помощью react-router-dom, использовала приватный роутинг для проверки
авторизации

PrivetAdel committed:
feat(redux): подключить Redux к проекту

Начала подключать редакс

PrivetAdel committed:
feat(redux): feat(redux): подключить Redux к проекту

Начала подключать редакс

bread-bread commented on src/App.js:
```diff
@@ -12,36 +13,17 @@ const useStyles = makeStyles({
+          {isLoggedIn && <Header />}
+          <PrivateRoute auth={isLoggedIn} exact path="/" component={() => <MapPage />} />
+          <PrivateRoute auth={isLoggedIn} path="/ProfilePage" component={() => <ProfilePage />} />
+          <Route auth={isLoggedIn} path="/LoginPage" component={() => <LoginPage />}  />
```
то что в роут передаешь auth в компонент LoginPage не попадает, поэтому просто достань с помощью useSelector значение в самой странице логина

PrivetAdel committed:
fix(components): исправить ошибки роутинга после подключения редакса

исправила ошибки роутинга после подключения редакса

PrivetAdel committed:
fix(redux): исправить тесты после подключения редакса

исправила часть тестов после подключения редакса

PrivetAdel committed:
feat(components): сверстать страницу профиля

Сверстала страницу профиля

PrivetAdel committed:
fix(components): feat(components): сверстать страницу профиля

Сверстала страницу профиля

PrivetAdel committed:
feat(pages): исправить роутинг, кэшировать данные в localstorage

исправила роутинг, закэшировала данные в localstorage, добавила страницу профиля

PrivetAdel committed:
test(redux): добавить тесты редьюсеров

протестировала редьюсеры
